src/base/projects/Browser.svelte

```diff
@@ -173,7 +173,7 @@
           {#if utils.isMarkdownPath(blob.path)}
             <Readme content={blob.content} />
           {:else}
-            <Blob {blob} />
+            <Blob line={browser.hash?.slice(2)} {blob} />
```

Comment:
we should check what we're parsing, this will pass in a line even with `#abcd`

Reply:
I could add a function that would do the slicing of the location hash here and  verify the result.
Although if you pass `#abcd` or anything else it just won't stop no harm done, not sure why we would need to parse and check the result

Reply:
The point is that currently `Blob` needs to handle a string, which could be anything, ie. not a number, whereas `Blob` only cares about line numbers at the moment, so it should accept the smallest possible input space that the type-checker can check, which is `number | null`. As a rule of thumb, you want to convert unstructured data into structured data as soon as possible, ie.  at the edges, to keep most of your code dealing with structured/typed data.

Reply:
Ok awesome! Understood, will provide a fix for it